src/OpenTelemetry/Metrics/MetricAggregators/IMetric.cs

```diff
@@ -16,21 +16,19 @@
 
 using System;
 using System.Collections.Generic;
-using System.Diagnostics.Metrics;
 
 namespace OpenTelemetry.Metrics
 {
-    // TODO: Need to determine what a Metric actually contains
-
-    internal struct Metric
+    internal interface IMetric
     {
-        internal readonly string Name;
-        internal IDataPoint Point;
+        string Name { get; }
+
+        DateTimeOffset StartTimeExclusive { get; }
+
+        DateTimeOffset EndTimeInclusive { get; }
+
+        KeyValuePair<string, object>[] Attributes { get; }
```

Comment:
I see usage of the term `Tag` in this PR which is consistent with `Activity.Tags` for trace data.

Question for metric data, do we want to continue to use `Tag` or do we want to align with the spec and call things `Attributes`?

Reply:
Maybe at least for the purposes of this PR we can standardize on the term `Tag` and defer the conversation of Tag vs. Attribute later.